src/Components/MainTableView/UpdateTuple/UpdateTuple.tsx

```diff
@@ -71,13 +71,83 @@ class UpdateTuple extends React.Component<{token: string, selectedSchemaName:str
     if (this.props.tupleToUpdate === prevProps.tupleToUpdate) {
       return;
     } else {
-      let tupleBuffer = Object.assign({}, this.state.tupleBuffer);
+      // let tupleBuffer = Object.assign({}, this.state.tupleBuffer);
+      // Object.values(this.props.tupleToUpdate).forEach((columns: any) => {
+      //   Object.entries(columns.primaryEntries).forEach((attributeKeyVal: any) => {
+      //     tupleBuffer[attributeKeyVal[0]] = attributeKeyVal[1]
+      //   })
+      //   Object.entries(columns.secondaryEntries).forEach((attributeKeyVal: any) => {
+      //     tupleBuffer[attributeKeyVal[0]] = attributeKeyVal[1]
+      //   })
+      // })
+      // this.setState({tupleBuffer: tupleBuffer});
```

Comment:
@mahos Was this intentionally left behind?

Reply:
oh good catch - no - I meant to remove once the new conversion worked. Will remove